autokeras/hypermodel/preprocessor.py

```diff
@@ -144,65 +144,156 @@ def set_weights(self, weights):
 class TextToNgramVector(base.Preprocessor):
     """Convert raw texts to n-gram vectors."""
 
-    def __init__(self, **kwargs):
+    def __init__(self, ngram_range=(1, 1), stop_words=None, **kwargs):
```

Comment:
You can follow the TextToIntSequence to add docstrings to this class to describe the two newly added args.